src/d3d9/d3d9_device.cpp

```diff
@@ -7640,14 +7640,20 @@ namespace dxvk {
     const D3D9_VK_FORMAT_MAPPING srcFormatInfo = LookupFormat(srcDesc->Format);
     const D3D9_VK_FORMAT_MAPPING dstFormatInfo = LookupFormat(dstDesc->Format);
 
-    const VkImageSubresource dstSubresource =
+    VkImageSubresource dstSubresource =
       dstTextureInfo->GetSubresourceFromIndex(
         dstFormatInfo.Aspect, 0);
 
-    const VkImageSubresource srcSubresource =
+    VkImageSubresource srcSubresource =
       srcTextureInfo->GetSubresourceFromIndex(
         srcFormatInfo.Aspect, src->GetSubresource());
 
+    if ((dstSubresource.aspectMask & srcSubresource.aspectMask) != 0) {
```

Comment:
Can this ever be 0, i.e. are stencil-only formats a thing in D3D9?

Reply:
I was worried about Depth -> Color like D32F -> R32F. I don't know whether that's allowed in RESZ, the docs are pretty much nonexistent. I *think* it should probably work as long as we don't pass 0 as the aspect masks to the backend.

Reply:
Oh and yes, stencil only formats are a thing: https://github.com/doitsujin/dxvk/blob/ca3aa2014b2f7252d84043e20d6e349ff849a786/src/d3d9/d3d9_format.cpp#L275

Should we explicitly error out int hat case?

Reply:
Fair point about color dst, I *think* the backend supports that.

I guess we should error out if the ANDed mask is 0 and dst is not color.